CactbotOverlay/CactbotOverlay.csproj

```diff
@@ -63,13 +63,11 @@
     <Reference Include="Newtonsoft.Json">
       <HintPath>ThirdParty\OverlayPlugin\Newtonsoft.Json.dll</HintPath>
     </Reference>
-    <Reference Include="OverlayPlugin">
-      <HintPath>ThirdParty\OverlayPlugin\OverlayPlugin.dll</HintPath>
-    </Reference>
     <Reference Include="OverlayPlugin.Common">
       <HintPath>ThirdParty\OverlayPlugin\OverlayPlugin.Common.dll</HintPath>
     </Reference>
-    <Reference Include="OverlayPlugin.Core">
+    <Reference Include="OverlayPlugin.Core, Version=0.4.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
For my own education: what's happening here and why?
```diff
-<Reference Include="OverlayPlugin.Core">
+<Reference Include="OverlayPlugin.Core, Version=0.4.0.0, Culture=neutral, processorArchitecture=MSIL">
+<SpecificVersion>False</SpecificVersion>
```

As much as I can gather from [StackOverflow](https://stackoverflow.com/q/16578819), it looks like we wouldn't want to change this reference? But, also as far as I can tell, it ultimately doesn't matter.

Reply:
I remember changing `SpecificVersion` to `false` since that made sense to me; we don't want to depend on a specific version of OverlayPlugin but as the StackOverflow answer pointed out, both versions should have an identical effect.